app/views/shared/_display_ad.html.erb

```diff
@@ -16,6 +16,18 @@
       </div>
     </div>
   </div>
+<% elsif display_ad.placement_area.start_with?("home_") %>
```

Comment:
(nitpick): 

In the case of home feed we have the condition 
` display_ad.placement_area.start_with?("feed_")` because we have multiple placement areas that can start with `feed`

However in this case we only have one placement area starting with home so perhaps we could rephrase condition to be more specific: 

```suggestion
<% elsif display_ad.placement_area == "home_hero" %>
```

I personally like to be more specific until we need to be general to avoid unexpected edge cases. However, its unlikely to occur here so you can use or discard this comment.